### DISCUSSION TITLE
triggering templator from an image and other apps

### DISCUSSION THREAD

**Discussion by dandobos** [SilentVoid13/Templater#849] [Help]
Hi... 

I use what is often referred to as a meta template to trigger different templates eg if it includes
- (j) in title then trigger journal template
- (PN) in title then trigger Permanent Note template

Here is the meta template:
[Base Note (template).md](https://github.com/SilentVoid13/Templater/files/9691770/Base.Note.template.md)

I have 2 challenges
1. I want to trigger a template from apple shortcuts or drafts but when I include eg (j) it does not do so
2. Whenever I share an image with obsidian, I want to trigger another template

I'm wondering if there is some way to do this? Or a workaround?

Thank you!

**Comment by AB1908** [comment]
This is somewhat outside the scope of Templater (or at least that's how I feel) because it depends on how you're sharing to Obsidian. I don't think the Obsidian API exposes any of this information to really work with.
